src/libmongoc/doc/learn/index.rst

```diff
@@ -0,0 +1,11 @@
+###########################
+Getting Started & Tutorials
```

Comment:
```suggestion
Tutorials
```
Suggest keeping titles simple and consistent with TDS terminology ("Tutorials", "How-To Guides", "Explanation", "Reference").